infra/aws/terraform/prow-build-cluster/hack/flux-update.bash

```diff
@@ -0,0 +1,85 @@
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/usr/bin/env bash
+set -xeuo pipefail
+
+# TODO(pkprzekwas): point at k8s.io main
+github_org=pkprzekwas
```

Comment:
Should this be changed before or after merging?

Reply:
This should be updated after we merge the PR

Reply:
What if we update this, merge the PR, and then deploy it to prod? I think that should be fine as well considering we already tested this on canary.

Reply:
Okay, I realized that's going to break canary. Let's update this after merging.